spec/controllers/account/dashboard_controller_spec.rb

```diff
@@ -1,5 +1,17 @@
 require 'rails_helper'
 
 RSpec.describe Account::DashboardController, type: :controller do
+  let!(:user) { create(:user) }
 
+  before do
+    sign_in user
+  end
+
+  context 'GET /account/dashboard' do
+      it 'should show index page' do
+        get :index
+        expect(response).to be_successful
```

Comment:
expect(response).to be_successful
        expect(response).to have_http_status(200)

doing the same 
leave only 1 line